code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -595,7 +522,39 @@ client/proc/one_click_antag()
 	else
 		return
 
+/datum/admins/proc/getCandidates(var/Question, var/jobbanType, var/datum/game_mode/nuclear/gametypeCheck)

```

Comment:
this third option should work for any game mode so that get candidates can check the age requirement of any antag type, not just nuke